cdap-runtime-ext-dataproc/src/main/java/co/cask/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -64,7 +64,7 @@
   // keys cannot be empty
   private static final Pattern LABEL_KEY_PATTERN = Pattern.compile("^[a-z][a-z0-9_-]{0,62}$");
   private static final Pattern LABEL_VAL_PATTERN = Pattern.compile("^[a-z0-9_-]{0,63}$");
-  private Map<String, String> systemLabels;
+  private Map<String, String> systemLabels = new HashMap<>();
```

Comment:
can createCluster be called before initialize()?

Reply:
nope. I added this because I had an older sdk (that didn't call initialize) and didn't want to rebuilt everything. Will remove this.